src/lib/logging/index.ts

```
@@ -0,0 +1,198 @@
+  type LogLevelType,
+} from "./types.js";
+
+export class FileLogger {
+  protected options: LoggingOptions;
+  protected formatter!: Formatter;
+  protected file: string;
+
+  constructor(file: string, options: LoggingOptions) {
+    this.options = options;
+    this.file = file;
+  }
```

Biome (2.3.13)

[error] Missing accessibility modifier on this member.

Use public to explicitly make a member public.

(lint/style/useConsistentMemberAccessibility)